scale-ci-watcher-env.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
 
-export JJB_CONFIG_PATH=""
-export SCALE_CI_WATCHER_REPO=""
-export SCALE_CI_WATCHER_REPO_BRANCH=""
-export SCALE_CI_WATCHER_REPO_PATH=""
-export WORKDIR=""
+# path to the jenkins job builder config to connect to jenkins
+export JJB_CONFIG_PATH="/root/scale-ci-pipeline/config/jjb.ini"
+# repo hosting scale-ci watcher
+export SCALE_CI_WATCHER_REPO="https://github.com/openshift-scale/scale-ci-pipeline.git"
+# branch to clone
+export SCALE_CI_WATCHER_REPO_BRANCH="master"
+# location of the scale-ci-watcher repo
+export SCALE_CI_WATCHER_REPO_PATH="/root/scale-ci-pipline"
+# location of the scale-ci-watcher repo
+export WORKDIR="/root/scale-ci-pipeline"
+# whether to update the scale-ci jobs or not
 export UPDATE_SCALE_CI_JOBS=true
+# This is useful when running the watcher job in jenkins, it adds any public key to the jump host, this way other teams can access the jump host with having the perf key.
 export ADD_PUBLIC_KEY=false
 export PUBLIC_KEY=""
```

Comment:
Good work adding comments to the new env vars, keep it consistent and add comments to all.